adapters/amqp-vertx/src/main/java/org/eclipse/hono/adapter/amqp/VertxBasedAmqpProtocolAdapter.java

```diff
@@ -734,15 +750,27 @@ protected void onCommandReceived(final ProtonSender sender, final CommandContext
         })
         .map(downstreamDelivery -> {
             context.accept();
+            metrics.incrementProcessedMessages(context.getEndpoint(), context.getTenantId());
```

Comment:
hmm, we also have the *hono.commands.response.delivered* metric which is supposed to track the number of responses being uploaded. However, it seems that we could as well simply use the *hono.messages.processed* metric for that purpose, we could then also record the response payload in *hono.messages.processed.payload* and get rid of *hono.commands.response.delivered* altogether.

Reply:
It sounds good to use the _hono.messages.processed.payload_ metric with type as  _command_ for the command response. This is already available and so I will remove the ` metrics.incrementProcessedMessages(context.getEndpoint(), context.getTenantId());`

Reply:
It sounds good to use the _hono.messages.processed_ metrics for command responses too. I will remove the _hono.commands.response.delivered_ part.

Reply:
:-) I will send a proposal to the mailing list.